crypto/serializer/serializer_local.h

```diff
@@ -32,6 +32,9 @@ struct ossl_serializer_ctx_st {
     OSSL_SERIALIZER *ser;
     void *serctx;
 
+    /* One of OSSL_KEYMGMT_WANT_PARAMETERS and OSSL_KEYMGMT_WANT_KEY */
```

Comment:
Artefact of translating *_WANT_* to *_SELECT_*

Reply:
Thanks!  Will fix